scripts/python_api_example_for_model_server.py

```diff
@@ -20,7 +20,7 @@
             "I want to learn how to play the piano.",
             ]
 
-model_name = "EleutherAI/gpt-j-6b"  # model_name from huggingface or local model path
```

Comment:
how do we maintain the availability of continuous batching? do we have set up related CI?

Reply:
1. we have `gpt-j` offline constinuous batching beam search generation check in pytest.
2. plan to add mlperf example and mark it as extension tests for accuracy and performance check (will discuss with @DDEle and @VincyZhang )

Reply:
+3. will consider adding datasets (like `lambada-openai`) accuracy check by using `continuous batching`

Reply:
https://github.com/intel/neural-speed/blob/main/tests/test_python_api.py#L111-L115